nbd/src/ImageInstance.cpp

```diff
@@ -31,7 +31,12 @@ namespace nbd {
 bool ImageInstance::Open() {
     int ret = 0;
 
-    ret = nebd_lib_init();
+    if (config_ && !config_->nebd_conf.empty()) {
+        ret = nebd_lib_init_with_conf(config_->nebd_conf.c_str());
+    } else {
+        ret = nebd_lib_init();
```

Comment:
why not use same function: nebd_lib_init(conf)? Handle conf within nebd_lib_init.

Reply:
functions in here are all C functions because thay are all decorated by extern "C", so thay aren't support default parameters and overloading.